### PR TITLE
Deprecate gpmapreduce

### PR DESCRIPTION
**Pull request by janebeckman**
Notices for deprecation of gpmapreduce and associated YAML file.

### CODE REVIEW

**Comment by Eulerizeit** [comment]
@ivannovick did we determine for sure we were deprecating this?

**Comment by ivannovick** [comment]
We are documeting in 5.0 it will removed in 6.0.  If we get pushback from customers, we may reverse our selves depending on the level and nature of the pushback.  This will effectively provide 1 years notice.

File: gpdb-doc/dita/ref_guide/yaml_spec.xml [hunk 1]
```diff
@@ -5,8 +5,10 @@
   <title id="ii20941">Greenplum MapReduce Specification</title>
   <body>
     <p>This specification describes the document format and schema for defining Greenplum MapReduce
-      jobs.</p>
-    <p><xref href="https://en.wikipedia.org/wiki/MapReduce" scope="external" format="html"
+      jobs. <note>Both the MapReduce tool (<codeph>gpmapreduce</codeph>) and its associated YAML
+        specification are deprecated and will be removed in the next major release (6.x) of
+        Greenplum Database. Use other query formats, such as SQL or PL/*, instead.</note></p>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/yaml_spec.xml]: Instead of PL/*, let's say "a procedural language extension".

**Comment by danielgustafsson** [comment]
Dibs on doing the actual removal when the day comes! =)

**Comment by Eulerizeit** [comment]
It can happen as soon as we create a 6.0-dev branch

On Thu, Mar 16, 2017 at 5:19 PM, Daniel Gustafsson <notifications@github.com

**Pull request merged**